src/megaapi_impl.cpp

```diff
@@ -16135,6 +16130,30 @@ int naturalsorting_compare (const char *i, const char *j)
     return 0;
 }
 
+void MegaApiImpl::sortNodes(std::vector<Node *> &nodes, int order)
+{
+    if (order == MegaApi::ORDER_NONE)
```

Comment:
Better to keep the existing logic and sort as per default: `ORDER_DEFAULT_ASC`. Simply remove this block ;)